replica/tablets.cc

```diff
@@ -49,6 +49,8 @@ schema_ptr make_tablets_schema() {
             .with_column("new_replicas", replica_set_type)
             .with_column("stage", utf8_type)
             .with_column("session", uuid_type)
+            .with_column("resize", utf8_type, column_kind::static_column)
+            .with_column("resize_generation", long_type, column_kind::static_column)
```

Comment:
update docs/dev/system_keyspace.md

Reply:
we already do in patch 'tablets: Add resize decision metadata to tablet metadata'

Reply:
ok